sentinelhub/config.py

```diff
@@ -46,6 +46,7 @@ class _SHConfig:
     download_sleep_time: float = 5.0
     download_timeout_seconds: float = 120.0
     number_of_download_processes: int = 1
+    max_rate_limit_retries: int | None = None
```

Comment:
since this parameter covers any retriable error, it might as well be renamed to `max_retries`

Reply:
That is true, I have pushed a change for this.